src/accel/RZMapMagneticField.hh

```diff
@@ -0,0 +1,78 @@
+//----------------------------------*-C++-*----------------------------------//
+// Copyright 2023 UT-Battelle, LLC, and other Celeritas developers.
+// See the top-level COPYRIGHT file for details.
+// SPDX-License-Identifier: (Apache-2.0 OR MIT)
+//---------------------------------------------------------------------------//
+//! \file RZMapMagneticField.hh
+//---------------------------------------------------------------------------//
+#pragma once
+
+#include <memory>
+#include <G4MagneticField.hh>
+#include <globals.hh>
```

Comment:
This puts the units in the global namespace; maybe `<CLHEP/Units/SystemOfUnits.h>` instead?